spec/features/orders/user_cancels_order_spec.rb

```diff
@@ -14,22 +14,23 @@
         password_confirmation: "pass",
         role: 0
       )
-      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@user)
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
 
-      @mike = Merchant.create(name: "Mike's Print Shop", address: '123 Paper Rd.', city: 'Denver', state: 'CO', zip: 80203)
-      @paper = @mike.items.create(name: "Lined Paper", description: "Great for writing on!", price: 20, image: "https://cdn.vertex42.com/WordTemplates/images/printable-lined-paper-wide-ruled.png", inventory: 3)
-      @pencil = @mike.items.create(name: "Yellow Pencil", description: "You can write on paper with it!", price: 2, image: "https://images-na.ssl-images-amazon.com/images/I/31BlVr01izL._SX425_.jpg", inventory: 100)
-      @order_1 = @user.orders.create(name: "Reg", address: "123 Street", city: "Denver", state: "CO", zip: "80202", user_id: @user.id)
-      ItemOrder.create(order_id: @order_1.id, item_id: @paper.id, price: 20, quantity: 2)
-      ItemOrder.create(order_id: @order_1.id, item_id: @pencil.id, price: 2, quantity: 2)
+      mike = Merchant.create(name: "Mike's Print Shop", address: '123 Paper Rd.', city: 'Denver', state: 'CO', zip: 80203)
+      paper = mike.items.create(name: "Lined Paper", description: "Great for writing on!", price: 20, image: "https://cdn.vertex42.com/WordTemplates/images/printable-lined-paper-wide-ruled.png", inventory: 3)
+      pencil = mike.items.create(name: "Yellow Pencil", description: "You can write on paper with it!", price: 2, image: "https://images-na.ssl-images-amazon.com/images/I/31BlVr01izL._SX425_.jpg", inventory: 100)
+      order_1 = user.orders.create(name: "Reg", address: "123 Street", city: "Denver", state: "CO", zip: "80202", user_id: user.id)
+      ItemOrder.create(order_id: order_1.id, item_id: paper.id, price: 20, quantity: 2)
+      ItemOrder.create(order_id: order_1.id, item_id: pencil.id, price: 2, quantity: 2)
 
-      visit "/profile/orders/#{@order_1.id}"
+      visit "/profile/orders/#{order_1.id}"
       click_button "Cancel Order"
-
+     
       expect(current_path).to eq("/profile")
-      expect(page).to have_content("Your order is now cancelled")
-      expect(@order_1.status).to eq("Cancelled")
+      expect(page).to have_content("Your order has been cancelled")
+
+      updated_order = Order.find(order_1.id)
```

Comment:
This was tough! Good thinking having the test make a database call vs. checking an existing variable